io/pci/dlpar.py

```diff
@@ -213,15 +213,18 @@ def test_drmgr_pci(self):
         '''
         drmgr remove, add and replace operations
         '''
-        for pci in self.pci_device:
-            self.set_adapter_details(pci)
-            for _ in range(self.num_of_dlpar):
-                self.do_drmgr_pci('r')
-                self.do_drmgr_pci('a')
-                self.validation_in_os(pci)
-            for _ in range(self.num_of_dlpar):
-                self.do_drmgr_pci('R')
-                self.validation_in_os(pci)
+        if self.sriov == "no":
+            for pci in self.pci_device:
+                self.set_adapter_details(pci)
+                for _ in range(self.num_of_dlpar):
+                    self.do_drmgr_pci('r')
+                    self.do_drmgr_pci('a')
+                    self.validation_in_os(pci)
+                for _ in range(self.num_of_dlpar):
+                    self.do_drmgr_pci('R')
+                    self.validation_in_os(pci)
+        else:
+            self.cancel("drmgr -c pci test is not supported for this device")
```

Comment:
@vaishnavibhat can you just add only following lines at the beginning of test, instead of changing the complete block.

```
if self.sriov == "yes":
    self.cancel("drmgr -c pci test is not supported for this device")

```